ZDF_2_CSV.py

```diff
@@ -0,0 +1,52 @@
+"""
+This example shows how to convert a Zivid point cloud from a .ZDF file format
```

Comment:
This regards all the files:
Is it obvious that this file is a sample file? Perhaps you can simplify the description to something like:
Convert ZDF point cloud to CSV format